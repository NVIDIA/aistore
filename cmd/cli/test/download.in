ais bucket create ais://$BUCKET // IGNORE
ais job rm download --all // IGNORE
ais job start download gs://lpr-vision/imagenet/imagenet_train-{000000..000140}.tgz ais://$BUCKET // FAIL "too many arguments"

ais job start download gs://minikube/iso/minikube-v0.23.2.iso.sha256 ais://$BUCKET --desc="minikube" // IGNORE
ais job wait download $(ais show job download --regex="minikube" --all | awk 'NR==2 {print $1}')
ais show job download --regex="minikube" // IGNORE
ais show job download $(ais show job download --regex="minikube" --all | awk 'NR==2 {print $1}')

ais job start download "gs://minikube/iso/minikube-v0.23.{2..4}.iso.sha256" ais://$BUCKET --desc="minikube-range" // IGNORE
ais job wait download $(ais show job download --regex="minikube-range" --all | awk 'NR==2 {print $1}')
ais show job download --regex="minikube" // IGNORE
ais show job download $(ais show job download --regex="minikube-range" --all | awk 'NR==2 {print $1}')

echo "[\"iso/minikube-v0.25.0.iso.sha256\", \"iso/minikube-v0.25.1.iso.sha256\"]" > /tmp/ais_test_dl_list.txt
ais job start download gs://minikube ais://$BUCKET --object-list=/tmp/ais_test_dl_list.txt --desc="minikube-multi" // IGNORE
ais job wait download $(ais show job download --regex="minikube-multi" --all | awk 'NR==2 {print $1}')
ais show job download --regex="minikube" // IGNORE
ais show job download $(ais show job download --regex="minikube-multi" --all | awk 'NR==2 {print $1}')
