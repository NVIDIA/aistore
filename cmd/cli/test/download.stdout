
^download.*finished$
Done: 1 file downloaded

^download.*finished$
Done: 3 files downloaded, skipped: 1

^download.*finished$
Done: 2 files downloaded
